fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/api/object/builder/v1/entity/FabricEntityType.java

```diff
@@ -51,6 +51,16 @@ default EntityType.Builder<T> alwaysUpdateVelocity(boolean alwaysUpdateVelocity)
 			throw new AssertionError("Implemented in Mixin");
 		}
 
+		/**
+		 * Sets whether the entity is able to execute commands.
+		 *
+		 * @param canPotentiallyExecuteCommands whether the entity is able to execute commands
+		 * @return this builder
+		 */
+		default EntityType.Builder<T> canPotentiallyExecuteCommands(boolean canPotentiallyExecuteCommands) {
```

Comment:
The method in the entity type builder takes a `Boolean` because the other method in this interface also takes a `Boolean`. I suppose the block entity type builder can also take this parameter.

Reply:
It does feel a little odd to add two diffrent methods that should do the same thing. I dont have a peference either way but I think it would be good for them to match.

Reply:
Ive added the bool to the block entity type builder.